aci/convert/convert.go

```diff
@@ -237,6 +275,35 @@ func (s serviceConfigAciHelper) getResourceRequestsLimits() (*containerinstance.
 	return &resources, nil
 }
 
+func (s serviceConfigAciHelper) getLivenessProbe() *containerinstance.ContainerProbe {
+	if s.HealthCheck != nil && !s.HealthCheck.Disable && len(s.HealthCheck.Test) > 0 {
+		testArray := s.HealthCheck.Test
+		switch s.HealthCheck.Test[0] {
+		case "NONE", "CMD", "CMD-SHELL":
+			testArray = s.HealthCheck.Test[1:]
+		}
+		if len(testArray) == 0 {
+			return nil
+		}
+
+		var retries int32 = 10
```

Comment:
magic number 10, is this a docker default ? Maybe use a const with a clear name for this

Reply:
This is completely arbitrary. I was also thinking about just letting it as `nil` if not explicitly set. WDYT @gtardif ?

Reply:
I would let the ACI default apply and not specify anything if the user does not